### PR TITLE
QT - Make classroom toggles accessible in channels tree

### PR DESCRIPTION
**Pull request by Oreonan**

### CODE REVIEW

File: Client/qtTeamTalk/channelstree.cpp [hunk 1]
```diff
@@ -1489,8 +1494,19 @@ bool ChannelsTree::eventFilter(QObject *object, QEvent *event)
                 {
                     item->setExpanded(true);
                 }
-                return true;
+                else if ((item->type() & USER_TYPE))
+                {
+                    QTreeWidgetItem* parent = item->parent();
+                    int parentid = (parent->data(COLUMN_ITEM, Qt::UserRole).toInt() & ID_MASK);
+                    UserRights rights = TT_GetMyUserRights(ttInst);
+                    if (TT_IsChannelOperator(ttInst, TT_GetMyUserID(ttInst), parentid) ||
```
Comment (bear101) [inline Client/qtTeamTalk/channelstree.cpp:1502]: Shouldn't non-operator/admin also be allowed to see if they can transmit?

Comment (Oreonan) [reply Client/qtTeamTalk/channelstree.cpp:1502]: Hmmm, maybe. But text of accessible text shouldn't be different in this case?

Comment (bear101) [reply Client/qtTeamTalk/channelstree.cpp:1502]: Doesn't have to be. Does it?

Comment (bear101) [reply Client/qtTeamTalk/channelstree.cpp:1502]: hm, maybe it should be "User can transmit..."

Comment (Oreonan) [reply Client/qtTeamTalk/channelstree.cpp:1502]: I see 2 solutions:
- We change text for all case to "User can transmit: yes/no"
- We don't change current text for operators/admins but we change for users by "transmission: allowed/disallowed"

**Comment by bear101** [comment]
I've removed the event filter and instead also put accessible text on channel column. Can you manage to use it the way it is now?

**Comment by Oreonan** [comment]
Yes it's OK for me.

**Pull request merged**